OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix membership

# Description
Processed a meaningless folde, updated lint ignore file .

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. check all the function is normal
